### ISSUE TITLE
LV2 build?

### ISSUE THREAD

**Issue by mxmilkiib** [edgjj/vitOTT#1]
Hi! I'm wondering if an LV2 release might be possible in the future? Thanks!

**Comment by edgjj** [comment]
Definitely! I have plans on migrating to JUCE 7, this 'd make LV2 possible.
